src/App.tsx

```diff
@@ -1,61 +1,131 @@
 import './App.scss';
+import React, { useState } from 'react';
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+function getUser(userId: number): User | null {
```

Comment:
don't forget about past checkLists
```suggestion
function getUserById(id: number): User | null {
```